extensions/api/data-management/asset/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/asset/AssetController.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2022 Diego Gomez
```

Comment:
As far as I know the Copyright should go to ZF, at least that is my latest information.